### PR TITLE
Add workshop notes

### PR DESCRIPTION
**Pull request by alexgleith**
### Proposed changes
Add a few brief notes about how to run through the workshop.

### Checklist (replace `[ ]` with `[x]` to check off)
- [x] Notebook created using the [DEA-notebooks template](https://github.com/GeoscienceAustralia/dea-notebooks/tree/develop)
- [x] Remove any unused Python packages from `Load packages`
- [x] Remove any unused/empty code cells
- [x] Remove any guidance cells (e.g. `General advice`)
- [x] Ensure that all code cells follow the [PEP8 standard](https://www.python.org/dev/peps/pep-0008/) for code. The `jupyterlab_code_formatter` tool can be used to format code cells to a consistent style: select each code cell, then click `Edit` and then one of the `Apply X Formatter` options (`YAPF` or `Black` are recommended).
- [x] Include relevant tags in the final notebook cell (refer to the [DEA Tags Index](https://docs.dea.ga.gov.au/genindex.html), and re-use tags if possible)
- [x] Clear all outputs, run notebook from start to finish, and save the notebook in the state where all cells have been sequentially evaluated
- [x] Test notebook on both the `NCI` and `DEA Sandbox` (flag if not working as part of PR and ask for help to solve if needed)
- [x] If applicable, update the `Notebook currently compatible with the NCI|DEA Sandbox environment only` line below the notebook title to reflect the environments the notebook is compatible with

### CODE REVIEW

File: Beginners_guide/Guided_tutorial.ipynb [hunk 1]
```diff
@@ -213,4 +218,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```
Comment (alexgleith) [inline Beginners_guide/Guided_tutorial.ipynb:221]: Does this bother anyone?

Comment (MatthewJA) [reply Beginners_guide/Guided_tutorial.ipynb:221]: No, since it's an autogenerated file.

**Review by MatthewJA — APPROVED** [review]
This looks good! Please "squash and merge" because the git history on this branch is a bit odd.

**Comment by robbibt** [comment]
> This looks good! Please "squash and merge" because the git history on this branch is a bit odd.

Yeah I think this branch derived from `stable` rather than `develop` - shouldn't be a major problem if it's squashed!

**Comment by alexgleith** [comment]
Nah, I'll fix it. Hang on.

**Comment by alexgleith** [comment]
Ok, clean commits now :-)

**Review by robbibt — APPROVED** [review]

**Pull request merged**